src/main/java/com/parasoft/demoapp/service/OrderService.java

```diff
@@ -41,6 +39,55 @@ public class OrderService {
     @Autowired
     private LocationService locationService;
 
+    @Transactional
+    public InventoryOperationRequestMessageDTO handleResponseQueue(InventoryOperationResultMessageDTO operationResult) {
```

Comment:
Please rename this method to be "handleMessageFromResponseQueue" to be consistent.